plugin-dev/Source/Sentry/Private/Android/SentrySubsystemAndroid.cpp

```diff
@@ -169,6 +169,32 @@ USentryId* SentrySubsystemAndroid::CaptureEventWithScope(USentryEvent* event, co
 	return SentryConvertorsAndroid::SentryIdToUnreal(*id);
 }
 
+USentryId* SentrySubsystemAndroid::CaptureException(const FString& type, const FString& message, int32 framesToSkip)
```

Comment:
I don't follow this. `CaptureException` does not actually capture an exception. Instead it adds a whole bunch of markers that are later getting picked up by the bridge?